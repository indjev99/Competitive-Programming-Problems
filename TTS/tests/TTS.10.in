100 10
757.84 -10000
-1251.24 10000
-10000 -8775.48
-10000 -5836.18
10000 -1904.39
-10000 8472.04
10000 2561.35
-10000 7402.61
-10000 9500.42
9144.17 -10000
-6973.74 -10000
-10000 -5002.54
-10000 -8238.79
3201.1 -10000
-4619.32 10000
-10000 7661.17
-10000 9299.48
-10000 9464.68
-999.44 10000
5201.69 -10000
-5674.94 -10000
-10000 6685.81
-10000 8220.61
10000 6221.44
-6803.89 -10000
-10000 2225.17
-10000 -7153.73
10000 4599.8
10000 2259.61
6392.87 10000
10000 -76.69
-2973.58 -10000
-8869.21 -10000
-3548.08 10000
5688.03 -10000
6534.37 10000
10000 -3597.94
6239.05 10000
-6454 -10000
-842.81 -10000
-10000 2538.85
-10000 -7955.49
3714.84 -10000
5330.35 10000
6678.41 -10000
-9298.63 -10000
-4158 10000
-1609.68 -10000
-5117.19 10000
10000 -6645.55
-10000 1034.83
6738.27 10000
2622.54 10000
1528.01 10000
1439.51 10000
-9716.67 10000
4630.46 10000
10000 325.43
-10000 -2085.85
-10000 -4351.11
4333.5 -10000
-10000 9895.75
10000 5292.61
10000 2756.89
10000 -9025.25
-10000 1961.69
-10000 -7282.83
-10000 -745.23
-520.38 10000
520.37 10000
10000 -8948.67
7227.61 10000
10000 -8396.58
-10000 -4374.09
-4366.08 10000
9152.7 -10000
3796.96 -10000
-9994.24 10000
-10000 -3921.04
-4311.73 -10000
-10000 -8679.39
-9178.17 -10000
1527.32 10000
2683.67 -10000
10000 -519.87
-5551.6 -10000
2081.47 -10000
2810.36 10000
-3570.3 -10000
-10000 -5426.56
-10000 7886.15
10000 -8589.91
-10000 1510.36
360.14 10000
-3772.98 10000
10000 798.98
2809.95 10000
3658.6 -10000
6765.22 -10000
1472.74 10000
-2097.89 -7818.01
7714.21 7132.75
4930.68 -1204.23
1482.03 -949.45
2329.78 -1419.44
4029.83 -3659.31
4797.31 8649.79
5223.61 -8114.22
7986.69 -4380.07
402.95 4705.68
