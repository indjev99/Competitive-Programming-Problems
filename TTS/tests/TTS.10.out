41257.127642
